### PR TITLE
Iteration5/past acttivities

### PR DESCRIPTION
**Pull request by JeffShepherd**
## Is this a fix or a feature? ⬇️

Feature

## What is the change? ⬇️

The user is now able to see their past activities, even upon page reload. Each past activity is represented by a card in the right side of the screen.

## Where should the reviewer start? ⬇️

Main.js line 36 evaluates a conditional, and then invokes the windowLoad function, which should be reviewed as well. Also, the new loadCard function should be reviewed. Activity.js also received updates to the saveToStorage method. 

## How should this be tested? ⬇️

In addition to a full regression test, the tester should ensure that past activities are displayed in the cards on the right side of the page after a refresh. Use the chrome dev tools to check that all properties are assigned to local storage correctly.

## Screenshot of functionality ⬇️
Screen Shot 2021-01-11 at 4 36 28 PM

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -32,9 +32,19 @@ var startNewActivityForm = document.querySelector('.start-new-activity')
...
+
+window.addEventListener('load', function() {
+  if (localStorage.length === 0) {
+    return
+  } else {
+    windowLoad()
+    addClass(defaultMessage)
+  }
+})
+
```
Comment (itsnameissteven) [inline main.js:37]: Nice! I noticed that when you add enough activities we get white space at the bottom of the page

File: main.js [hunk 2]
```diff
@@ -150,7 +162,34 @@ function createNewActivity() {
...
+        <p class="card-description">${thing[i].description}</p>
+      </div>
+      <div class="card-section">
+      <button class="card-category-indicator ${color}"type="button" name="button"></button>
+      </div>
+    </article>`
   }
-}
\ No newline at end of file
+}
```
Comment (itsnameissteven) [inline main.js:172]: I wonder if we can make this more dynamic!

File: main.js (hunk 1 above)
Comment (itsnameissteven) [inline main.js:37]: do we need an anon function here???

File: main.js [hunk 3]
```diff
@@ -129,19 +139,21 @@ function displayCard() {
...
+        <p class="card-description">${currentActivity.description}</p>
       </div>
       <div class="card-section">
       <button class="card-category-indicator ${color}"type="button" name="button"></button>
       </div>
     </article>`
-}
+};
 
 function createNewActivity() {
```
Comment (itsnameissteven) [inline main.js]: can this whole function be run by location.reload()

**Review by itsnameissteven — APPROVED** [review]
Thanks for making those changes. Everything is working well now!

File: main.js (hunk 1 above)
Comment (whitneywilkes) [reply main.js:37]: Great thinking on testing that design component!

**Pull request merged**